tesseract_process_managers/include/tesseract_process_managers/core/process_planning_request.h

```diff
@@ -39,9 +39,15 @@ TESSERACT_COMMON_IGNORE_WARNINGS_POP
 
 namespace tesseract_planning
 {
+/** @brief The name of the primary executor (thread pool) */
+static const std::string PRIMARY_EXECUTOR_NAME = "primary";
+
 class ProcessPlanningRequest
 {
 public:
+  /** @brief The name of the executor to use */
+  std::string executor{ PRIMARY_EXECUTOR_NAME };
```

Comment:
Maybe executor_name? I was a bit confused when I saw in the cpp that you were doing executor==rhs.executor since I didn't realize it was actually the name not the executor itself.